cookbooks/continental_extension/continental_extension.prm

```diff
@@ -7,92 +7,144 @@
 #      Depth-dependent extension, two-stage breakup and cratonic underplating at rifted margins
 #   3. Naliboff, J., and Buiter, S.H. (2015), Earth Planet. Sci. Lett., v.421, p.58-67,
 #      "Rift Reactivation and migration during multiphase extension"
+#   4. Naliboff, J., Glerum, A., Sascha, S., Peron-Pinvidic, G., and Wrona, T. (2020), Geophys.
+#      Res. Lett., 47, e2019GL086611, "Development of 3‐D rift heterogeneity through fault
+#      network evolution"
+#   5. Sandiford, D., Brune, S., Glerum, A., Naliboff, J., and Whittaker, J.M. (2021), Geophys.
+#      Geochem. Geosys., 22, e2021GC009681, "Kinematics of footwall exhumation at oceanic
+#      detachment faults: Solid-block rotation and apparent unbending"
 
 ####  Global parameters
 set Dimension                              = 2
 set Start time                             = 0
 set End time                               = 5e6
 set Use years in output instead of seconds = true
-set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Nonlinear solver scheme                = single Advection, iterated defect correction Stokes
 set Nonlinear solver tolerance             = 1e-4
-set Max nonlinear iterations               = 10
+set Max nonlinear iterations               = 100
```

Comment:
Do you actually need this?

Reply:
I prefer this over silently continuing after 10 iterations...

Reply:
Yes, on the first time step it requires quite a few nonlinear iterations (<5 after that).